SecretManager/.OwlBot.yaml

```diff
@@ -1,6 +1,6 @@
 deep-copy-regex:
     - source: /google/cloud/secrets/(v1beta1)/.*-php/(.*)
       dest: /owl-bot-staging/SecretManager/$1/$2
-    - source: /google/cloud/secretmanager/(v1)/.*-php/(.*)
+    - source: /google/cloud/secretmanager/(v1|v1beta2)/.*-php/(.*)
```

Comment:
Hope the reason behind a v1beta2 after v1 release is sorted out by @saranshdhingra

Reply:
Not yet, discussion is going on.
Won't merge this until I have an answer.

Reply:
It's been answered that v1 beta 2 is there for new secret manager apis released in beta mode post which they would get transferred into stable channel.